meshtastic/deviceonly.proto

```diff
@@ -148,6 +152,11 @@ message NodeInfoLite {
    * Persists between NodeDB internal clean ups
    */
   bool is_favorite = 10;
+
+ /*
+  * maximum initial hop_limit value specified by this node, only present for nodes running new firmware
```

Comment:
This is being initially implemented to allow us to know if `hops_away` values are valid, but this could potentially be a valid way to see what users are sending very high hop_limits from their nodes.